homeassistant/components/goodwe/sensor.py

```diff
@@ -109,6 +112,18 @@ class GoodweSensorEntityDescription(SensorEntityDescription):
         value=lambda prev, val: prev if not val else val,
         available=lambda entity: entity.coordinator.data is not None,
     ),
+    "VA": GoodweSensorEntityDescription(
+        key="VA",
+        device_class=SensorDeviceClass.APPARENT_POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfApparentPower.VOLT_AMPERE,
+    ),
+    "var": GoodweSensorEntityDescription(
+        key="var",
+        device_class=SensorDeviceClass.REACTIVE_POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=POWER_VOLT_AMPERE_REACTIVE,
+    ),
```

Comment:
Shouldn't these be disabled by default?

Reply:
Why ? Thery are valid sensor, already being provided, now just with proper unit/device_class.

Reply:
As per our developer docuimentation:

![image](https://user-images.githubusercontent.com/195327/222967524-f1acbf7e-9a46-4af4-983a-7758575be1d4.png)

Let's disable these by default.

Reply:
As mentioned above, but maybe not completely understood - this change of code does NOT add any new sensors.
Those sensors are present (and enabled) since the integration was accepted >1 year ago.
(And the ticket name adjustment done by MartinHjelmare, is not really proper one, the original title is more precise).

The only change here is that since HA core was extended with VA and var units recently, these sensors are now categorized with APPARENT_POWER and REACTIVE_POWER device classes.
Again, the sensors are already there, but before this change, they were set up as plain MEASUREMENT sensors w/o device class and with their _attr_native_unit_of_measurement explicitly set - see lines 208-210.

Reply:
> this change of code does NOT add any new sensors.

It does add new entity descriptions, which I review as such.

> Those sensors are present (and enabled) since the integration was accepted >1 year ago.

Not sure how that matters?

> Again, the sensors are already there,

Ok, in that case, consider it a review comment on the existing sensor 🤷‍♂️

Reply:
So you are suggesting to create a breaking change and make some sensors now disabled by default ?

Reply:
That is not a breaking change, as it will not affect existing installations only new ones.

../Frenck

Reply:
I see, I did not realize that.
So I will make the VA/var sensors disabled by default (for new installations).